docs/BasicCommunication/OnSystemTimeReady/index.md

```diff
@@ -0,0 +1,37 @@
+## OnSystemTimeReady
+
+Type
+: Notification
+
+Sender
+: HMI
+
+Purpose
+: Inform SDL about readiness to provide system time
+
+!!! must
+
+The HMI must send BC.OnSystemTimeReady to SDL due to new ignition cycle in order to notify SDL about readiness to provide system time.
```

Comment:
The HMI must send BC.OnSystemTimeReady to SDL due to new ignition cycle in order to notify SDL about readiness to provide system time.
to
The HMI must send a BC.OnSystemTimeReady notification to SDL when a new ignition cycle occurs in order to notify SDL that the HMI is once again ready to provide accurate system time.

Reply:
@mrapitis, thank you.
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/671b954b558288f41dc50414381000896037d493